src/main/java/org/fundacionjala/coding/william/CountChar.java

```diff
@@ -0,0 +1,21 @@
+package org.fundacionjala.coding.william;
+
+import java.util.stream.IntStream;
+
+/**
+ * Class CountChar.
+ */
+public class CountChar {
+
+    /**
+     * Method that performs the character count of a string.
+     *
+     * @param cadena input string.
+     * @param letra  character that we will compare.
+     * @return returns the number of characters that exist in a string.
+     */
+    public int countChar(final String cadena, char letra) {
+        int count = (int) IntStream.range(0, cadena.length()).filter(i -> cadena.toLowerCase().charAt(i) == letra).count();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "count". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)